docs/advanced.rst

```diff
@@ -153,6 +153,20 @@ input. Otherwise the process will hang forever!::
 
 Above example will install ``flask`` and a private package ``private-test-package`` from GCP.
 
+☤ Supplying additional arguments to pip
+------------------------------------------------
+
+There may be cases where you wish to supply additional arguments to pip to be used during the install phase.
+For example, you may want to enable the new pip experimental feature for using
```

Comment:
Suggestion in order to make this paragraph age well:

> For example, you may want to enable a new pip experimental feature ....

In a few months from now system-certificates won't be the new ...

Reply:
Good idea.